vhpidirect/quickstart/logicvector/main.c

```diff
@@ -20,34 +20,34 @@ char logic_vec_A[3];
 char logic_vec_B[6];
 
 int getLogicVecASize(){
```

Comment:
What about merging `getLogicVecASize` and `getLogicVecASize` into `getLogicVecSize(int id)` where id 0 is for A and id 1 is for B?

Reply:
I did try something like that... oh, I took it too far and tried to switch retur the 2 different vectors (of different bounded size, so VHDL didn't like it). I'll switch the size function!

Reply:
It's fun that you are kind of following the same steps, and mistakes, as I did: ghdl/ghdl#797.